package.json

```
@@ -13,6 +13,7 @@
   "private": true,
   "dependencies": {
     "@babel/standalone": "latest",
+    "acorn": "8.7.1",
```

```suggestion
    "acorn": "^8.7.1",
```
no dep anywhere should ever be pinned; that’s what lockfiles are for